Automattic/pocket-casts-android
# Display Year vs Year story

## Description

This adds the year vs year story UI.

Figma: lH66LwxxgG8btQ8NrM0ldx-fi-3054_3987

## Testing Instructions

1. Sign in with an account that listened to at least 5 minutes of episodes this year.
2. Sync your profile data.
3. Close the app.
4. Reopen the app.
5. Open Playback 2024.
6. You might see a loading progress bar and it might take some time to sync your data.
7. Once data is synced go to the year vs year story and verify it.

## Screenshots or Screencast 

### Preview

![Screenshot 2024-10-17 at 16 18 37](https://github.com/user-attachments/assets/260019f5-aa83-4ad5-be5c-a8a3c01ca915)

### Animation

https://github.com/user-attachments/assets/a7c3ffd5-2f41-49ec-accc-7def46bec1d4


## Checklist
 
#### I have tested any UI changes...
